sql/keys.go

```diff
@@ -57,7 +57,11 @@ var normalize = unicode.SpecialCase{
 // NormalizeName normalizes to lowercase and Unicode Normalization Form C
 // (NFC).
 func NormalizeName(name string) string {
-	return norm.NFC.String(strings.Map(normalize.ToLower, name))
+	lower := strings.Map(normalize.ToLower, name)
+	if !norm.NFC.IsNormalString(lower) {
```

Comment:
`IsNormalString` isn't simple: https://github.com/golang/text/blob/master/unicode/norm/normalize.go#L106. I wonder if it would be better to check some sort of quick `IsASCII` check.